dbms/src/Storages/DeltaMerge/DeltaMergeStore_InternalBg.cpp

```diff
@@ -249,9 +249,32 @@ bool DeltaMergeStore::handleBackgroundTask(bool heavy)
 
 namespace GC
 {
+enum Type
+{
+    Unknown,
+    TooManyDeleteRange,
+    TooLargeDTFileRange,
```

Comment:
I think `TooMuchOutOfRange` is better. The thing here is not too large range, but small range.

Reply:
Fixed.